source/change-streams.rst

```diff
@@ -320,8 +351,60 @@ Presently change streams support only a subset of available aggregation stages:
 
 A driver MUST NOT throw an exception if any unsupported stage is provided, but instead depend on the server to return an error.
 
-The aggregate helper method MUST have no new logic related to the ``$changeStream`` stage. Drivers MUST be capable of handling `TAILABLE_AWAIT <https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst#read>`_  cursors from the aggregate command in the same way they handle such cursors from find.
+The aggregate helper methods MUST have no new logic related to the ``$changeStream`` stage. Drivers MUST be capable of handling `TAILABLE_AWAIT <https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst#read>`_  cursors from the aggregate command in the same way they handle such cursors from find.
 
+``Collection.watch`` helper
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:since: 3.6
```

Comment:
Likewise, should this "since" be omitted?

Reply:
Done